tests/stack/test_cli.py

```diff
@@ -22,13 +22,31 @@
 # limitations under the License.
 # TODO(alinxie)Write test which deploys a stack and validates the status json.
 # pylint:disable=redefined-outer-name
+# pylint:disable=unused-argument
 
+import os
+import json
 import pytest
 import mock
 from click.testing import CliRunner
 
 import databricks_cli.stack.cli as cli
 from tests.utils import provide_conf
+from tests.stack.test_api import TEST_STATUS, TEST_STACK
+
+TEST_STACK_PATH = 'stack/stack.json'
+
+
+def _write_stack(tmpdir):
```

Comment:
To be more specific, `_save_test_stack_config`?

Reply:
maybe `_write_test_stack_config`? Since `save` implies that I have made changes to the stack.